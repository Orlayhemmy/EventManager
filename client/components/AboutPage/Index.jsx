import About from './Container/aboutPage';

export default About;
