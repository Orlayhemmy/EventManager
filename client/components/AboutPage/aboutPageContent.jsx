import React from 'react';

/**
 * @description AboutPageContent form component
 */
export default class AboutPageContent extends React.Component {
  /**
   * @memberof HomeContent
   * @method render
   * @description it renders the component
   * @returns the HTML of AboutPageContent component
   */
  render() {
    return (
      <div id="about" className="container">
        <div className="row">
          <div className="media-body bg">	<br/>
            <h1>ABOUT US</h1>
            <h2>What Ecenter Does</h2>
            <p>Imo State is a southeastern Nigerian State and, being an oil producing State, is also counted among the Niger Delta States. Its Capital city is Owerri. The State has an estimated population of about 4 million people and Igbo is the predominant tribe. Looking to book luxury event centes or affordable event centers in Imo? You can book them on Spacebook.ng. Be sure to compare event center prices in Imo and read reviews on the site to help you take advantage of discounted event center rates and other special offers available when you book using Spacebook.ng
            </p>
            <h2>Why Rovan</h2>  
        	 	<p>Today, Many people are hesitant to take their clothes to local Laundromats because 
              they fear that optimal care and hygiene may not be maintained and that their clothes 
              may either get discolored, torn or misplaced. Haggling with locals over their ever 
              changing prices is another pain. Everyone has a penchant for expensive clothes, with 
              exquisite designs and fabric, that makes them stand apart. Why should you then settle 
              even for the second best when it comes to laundering them?
            </p> 
             
          </div>       
        </div>
      </div>
    );
  }
}