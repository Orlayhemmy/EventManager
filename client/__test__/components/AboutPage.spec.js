import React from 'react';
import expect from 'expect';
import AboutPage from '../../components/AboutPage/Container/AboutPage';

describe('AboutPage Component', () => {
  const props = {
    location: {
      pathname: '/about'
    }
  };
  it('should render without error', () => {
    const wrapper = shallow(<AboutPage {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
