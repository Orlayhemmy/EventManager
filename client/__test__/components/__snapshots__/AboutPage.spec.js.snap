// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPage Component should render admin dashboard page 1`] = `
<div
  id="about-page"
>
  <Connect(NavBar) />
  <AboutPageContent />
  <Footer />
</div>
`;
